### PR TITLE
fix "error executing lua callback ... attempt to concatenate local 'method' (a table value)"

### PR DESCRIPTION
**Pull request by bartman**
After updating my plugins I his this error, while editing a shell script.  Other LSPs seem to work fine.

```
Error detected while processing InsertLeave Autocommands for "*":
Error executing lua callback: .../.local/share/nvim/lazy/none-ls.nvim/lua/null-ls/rpc.lua:131: attempt to concatenate local 'method' (a table value)
stack traceback:
        .../.local/share/nvim/lazy/none-ls.nvim/lua/null-ls/rpc.lua:131: in function 'notify'
        ...art/usr/neovim/share/nvim/runtime/lua/vim/lsp/client.lua:784: in function 'notify'
        ...ocal/share/nvim/lazy/none-ls.nvim/lua/null-ls/client.lua:205: in function 'notify_client'
        .../.local/share/nvim/lazy/none-ls.nvim/lua/null-ls/rpc.lua:149: in function 'flush'
        ....local/share/nvim/lazy/none-ls.nvim/lua/null-ls/init.lua:49: in function <....local/share/nvim/lazy/none-ls.nvim/lua/null-ls/init.lua:48>
```

It would seem that the `method` passed to `dispatchers.notify()` method is a table, not a string.

I have used `vim.inspect(method)` to convert that `method` table to a string before concatenation.

Problem went away, LSP working fine in bash.

### CODE REVIEW

**Comment by mochaaP** [comment]
I believe this is likely the same issue with #209.

Are you using nightly?

**Comment by mochaaP** [comment]
I could not reproduce this. Can you attach an example of the table in question?

**Review by mochaaP — CHANGES_REQUESTED** [review]
`method` MUST be a string. see `:h vim.lsp.rpc.start()`.

**Comment by jiangyinzuo** [comment]
https://github.com/nvimtools/none-ls.nvim/blob/8fd7b29648a48502f1aa72ffe0c19e56069258c6/lua/null-ls/client.lua#L204-L209

I met the same problem. I changed the line 205 `client:notify(method, params)` to `client.notify(method, params)`, then it works.

**Comment by mochaaP** [comment]
hi @jiangyinzuo, what neovim commit are you on?

**Comment by jiangyinzuo** [comment]
> hi @jiangyinzuo, what neovim commit are you on?

Oh, I updated to the latest commit and then no error occurred. Thanks.

**Comment by mochaaP** [comment]
tldr: update to latest nightly.

**Comment by bartman** [comment]
I updated nvim and none-ls.nvim to latest commits, and problem went away.

thank you.
